cmd/geth/main.go

```diff
@@ -420,6 +427,30 @@ func startNode(ctx *cli.Context, stack *node.Node, backend ethapi.Backend) {
 		}()
 	}
 
+	// Spawn a standalone goroutine to monitor the chain head for the l2 migration block.
+	// When the l2 migration block is reached, close the entire stack.
```

Comment:
Should we use a goroutine? Wouldn't the result be more predictable if we executed this without spawning s new goroutine?

Reply:
Removed this change